scan/src/components/modal/submitTx/DelegateMsg.re

```diff
@@ -0,0 +1,43 @@
+[@react.component]
+let make = (~setMsgsOpt) => {
+  let (validator, setValidator) = React.useState(_ => EnhanceTxInput.{text: "", value: None});
```

Comment:
Why dont we create `EnhanceTxInput.empty = {text: "", value: None}` so we can reuse?